qe_metrics/utils/product_utils.py

```diff
@@ -12,23 +13,50 @@
 LOGGER = get_logger(name=__name__)
 
 
+def fetch_config_file_content_from_url(base_url: str, file_name: str) -> requests.Response:
+    return requests.get(f"{base_url}/configs/{file_name}")
+
+
 def products_from_repository() -> Dict[str, Dict[str, str]]:
-    res = requests.get("https://raw.githubusercontent.com/RedHatQE/qe-metrics-products-config/main/product-config.yaml")
-    return yaml.safe_load(res.content.decode("utf-8"))
+    config_dict: Dict[str, Dict[str, str]] = {}
+    base_url = "https://raw.githubusercontent.com/RedHatQE/qe-metrics-products-config/main"
```

Comment:
Maybe in the config file? Probably won't change very frequently, so if you want to keep this here, I'm fine with it.

Reply:
I think we can leave it as is since this is static repo set for this tool only.